File: Models/Driver.cs
namespace DAFTech.DriverLicenseSystem.Api.Models
{
    public class Driver
    {
        public int DriverId { get; set; }
        public string LicenseId { get; set; }
        public string FullName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string LicenseType { get; set; }
        public DateTime ExpiryDate { get; set; }
        public string QRRawData { get; set; }
        public string OCRRawText { get; set; }
        public DateTime CreatedDate { get; set; }
        public int RegisteredBy { get; set; }
    }
}

File: mobile_flutter/lib/models/driver.dart
class Driver {
  final int driverId;
  final String licenseId;
  final String fullName;
  final DateTime dateOfBirth;
  final String licenseType;
  final DateTime expiryDate;
  final String qrRawData;
  final String ocrRawText;

  Driver({
    required this.driverId,
    required this.licenseId,
    required this.fullName,
    required this.dateOfBirth,
    required this.licenseType,
    required this.expiryDate,
    required this.qrRawData,
    required this.ocrRawText,
  });

  Map<String, dynamic> toJson() {
    return {
      'licenseId': licenseId,
      'fullName': fullName,
      'dateOfBirth': dateOfBirth.toIso8601String(),
      'licenseType': licenseType,
      'expiryDate': expiryDate.toIso8601String(),
      'qrRawData': qrRawData,
      'ocrRawText': ocrRawText,
    };
  }
}
